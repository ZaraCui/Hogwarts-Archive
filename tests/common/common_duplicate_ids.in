ADD COLLECTION data/books_small.csv
ADD STUDENT Harry Potter
COMMON 100000 100000
EXIT

