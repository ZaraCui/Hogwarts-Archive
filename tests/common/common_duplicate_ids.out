user: No such collection.

user: Success.

user: Duplicate students provided.

user: Ending Archive process.
